src/shogun/distributions/HMM.h

```diff
@@ -476,14 +521,22 @@ class CHMM : public CDistribution
 		 * @param model model which holds definitions of states to be learned + consts
 		 * @param PSEUDO Pseudo Value
 		 */
-
 		CHMM(
 			int32_t N, int32_t M, Model* model, float64_t PSEUDO);
+
+		/// @param obs observations
```

Comment:
All arguments should have Doxygen documentation. In this method and the ones below as well.